modules/porous_flow/src/actions/PorousFlowUnsaturated.C

```diff
@@ -161,6 +161,8 @@ PorousFlowUnsaturated::addKernels()
     InputParameters params = _factory.getValidParams(kernel_type);
     params.set<UserObjectName>("PorousFlowDictator") = _dictator_name;
     params.set<bool>("strain_at_nearest_qp") = _strain_at_nearest_qp;
+    if (_base_name != "")
```

Comment:
maybe `empty()`

Reply:
Hey @cpgr - thanks for the review.  I'm happy to use str.empty() of course, but i'm curious as to the difference between `== ""` and `.empty()`.  Can you tell me?

Reply:
I think they are probably the same thing, but I just like empty for readability! It certainly wouldn't make any difference to the working of the code, so feel free to ignore.

Reply:
thanks